sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/CaseInsensitiveMap.scala

```diff
@@ -23,16 +23,30 @@ package org.apache.spark.sql.catalyst.util
 class CaseInsensitiveMap(map: Map[String, String]) extends Map[String, String]
```

Comment:
Good question. For some reason I was hung up on making  only the case-sensitive key available to the caller. Changed the  code to expose the original map , it made code simpler. Thank you very much  for the suggestion.